code_samples/jaas_sample/README.md

```diff
@@ -0,0 +1,39 @@
+# JAAS Code Sample
+
+This code sample demonstrates how to assign different permission policies for different user groups with the Hazelcast's JAAS based security features.
+Security features used in this code sample requires **Hazelcast IMDG Enterprise** edition.
+
+---
+
+Users within the `adminGroup` have create, destroy, read and put permissions over the `importantAdminMap`.
+
+Users within the `readerGroup` have just create and read permissions over the `importantReaderMap`.
+
+To test the code sample,
+
+* Make sure [hazelcast.xml](hazelcast-member/src/main/resources/hazelcast.xml) is in your class path.
+
+* Start the Hazelcast member with [Bootstrap.java](hazelcast-member/src/main/java/com/company/Bootstrap.java).
```

Comment:
We need to give the full java command for starting the bootstrap, including jars. Including how to get Java if its not on their laptop.  Remember these are NOT Java developers, assume they know nothing.

Reply:
I added a seperate README under the hazelcast-member folder describing the steps to compile and run the Bootstrap application